internal/learn/contribute-to-wiki/getting_started/developer.md

```diff
@@ -7,15 +7,11 @@ image: /img/iota-wiki.png
 
 # Writing as a Dev
 
-Developers rarely revel in reading documentation, let alone in writing it. Writing documentation is hard, and it is even harder to write a captivating one. Luckily, IOTA has a team of dedicated technical writers. We can help you, and you can help us. This document explains how.
-
-You are the author of your code. It makes you the original source of knowledge, the most reliable and often the only one. Technical writers cannot replace you or let you forget about the documentation. Quite the opposite, we want you to always keep it in mind so we could support you and make sure that users and other developers enjoy learning about your work.
-
-This document will evolve as we invent more ways to help each other. Check for updates once in a while.
+As a developer, you are responsible for documentation of your project. You are the one to choose its content, write it, and keep it updated with every release of your software. The [wiki maintainers](https://wiki.iota.org/team) and technical writers from IOTA Foundation will help you to coordinate the work, setup your project, and bring it to the standard.
 
 ## Make a Friend
 
-If you have not gotten in a contact with a technical writer yet, do it! We can advise you, ask the right questions, and plan together. Reach out to us through the `#iota-wiki` Discord channel,the `#dev-documentation` Slack channel, Slack DMs or via e-mail — whatever you have access to.
+Before you start, contact a technical writer. We could then take a look at your project, assess its state, give you an advice, and plan the work together. Reach out to us through the `#iota-wiki` Discord channel,the `#dev-documentation` Slack channel, Slack DMs or via e-mail — whatever you have access to.
```

Comment:
```suggestion
Before you start, contact a technical writer. We could then take a look at your project, assess its state, give you advice, and plan the work together. Reach out to us through the `#iota-wiki` Discord channel, the `#dev-documentation` Slack channel, Slack DMs, or via e-mail.
```
"— whatever you have access to " doesn't seem to add much to the sentence and ends with a preposition.